recipes/chrpath/chrpath_0.16.oe

```diff
@@ -4,14 +4,19 @@
 ## http://linux.die.net/man/1/chrpath
 
 require ${PN}.inc
-SRC_URI = "http://alioth.debian.org/frs/download.php/file/3648/chrpath-0.14.tar.gz"
-SRC_URI += "file://standarddoc.patch"
-SRC_URI += "file://cross-endian.patch"
+SRC_URI = "https://alioth.debian.org/frs/download.php/latestfile/813/chrpath-0.16.tar.gz"
+#SRC_URI += "file://standarddoc.patch"
+#SRC_URI += "file://cross-endian.patch"
```

Comment:
What is your opinion of these two patches?  Are they still relevant, or should they go?

In any case, leaving them in here, but commented out in recipe is not the way to go...